packages/jasmine/package.json

```diff
@@ -15,6 +15,9 @@
         "jasmine",
         "bazel"
     ],
+    "bin": {
+        "jasmine_runner": "jasmine_runner.js"
```

Comment:
this will pollute the users `node_modules/.bin` folder with a strangely named program. I think it should be `bazel-jasmine-runner`

Reply:
Agreed.